dask/dataframe/tests/test_shuffle.py

```diff
@@ -1226,13 +1226,28 @@ def test_set_index_nan_partition():
 @pytest.mark.parametrize("ascending", [True, False])
 @pytest.mark.parametrize("by", ["a", "b"])
 @pytest.mark.parametrize("nelem", [10, 500])
-@pytest.mark.parametrize("nparts", [1, 10])
 def test_sort_values(nelem, nparts, by, ascending):
     np.random.seed(0)
     df = pd.DataFrame()
     df["a"] = np.ascontiguousarray(np.arange(nelem)[::-1])
     df["b"] = np.arange(100, nelem + 100)
-    ddf = dd.from_pandas(df, npartitions=nparts)
+    ddf = dd.from_pandas(df, npartitions=10)
+
+    with dask.config.set(scheduler="single-threaded"):
```

Comment:
Is this configuration important? I'd expect it to work regardless of the scheduler I use, and it doesn't actually seem to take effect, since we leave the config block before ever computing.

Reply:
This is here for debugging purposes, as sorting helper functions like `set_partitions_pre` run in parallel on a multi-threaded scheduler and it's not possible to debug each individual run of the function unless this config option is set.

You're correct in expecting this to work without the single-threaded scheduler, happy to remove if you think that's the best option here

Reply:
I don't have a strong opinion on whether it should stay, though maybe a comment that it's only here for debugging purposes would be helpful